src/Microsoft.CodeQuality.Analyzers/UnitTests/QualityGuidelines/MarkMembersAsStaticTests.Fixer.cs

```diff
@@ -29,5 +30,1144 @@ protected override CodeFixProvider GetCSharpCodeFixProvider()
         {
             return new CSharpMarkMembersAsStaticFixer();
         }
+
+        [Fact]
```

Comment:
Could you please add some tests for cross language references?

Reply:
Ah, that is indeed a big hole. Let me file a separate issue as we have no test infrastructure support for it in the repo.

Reply:
For now, I am going to just skip updating references if the language of the referenced node is different - so you will get explicit build errors in referencing projects of different language.

Reply:
Filed https://github.com/dotnet/roslyn-analyzers/issues/1986 to track this work